internal/handlers/pg/pjson/pjson.go

```diff
@@ -72,7 +90,10 @@ type pjsontype interface {
 func checkConsumed(dec *json.Decoder, r *bytes.Reader) error {
 	if dr := dec.Buffered().(*bytes.Reader); dr.Len() != 0 {
 		b, _ := io.ReadAll(dr)
-		return lazyerrors.Errorf("%d bytes remains in the decoded: %s", dr.Len(), b)
```

Comment:
That was a bug - `dr.Len()` is zero in this case, as after `ReadAll`, nothing is left there.